### PR TITLE
MGMT 9312: UI add creation date to infraenv list

### PR DESCRIPTION
**Pull request by ammont82**
Related to https://issues.redhat.com/browse/MGMT-9312

Depends to previous PR: https://github.com/stolostron/console/pull/1469

### CODE REVIEW

**Comment by jelkosz** [comment]
/hold
this is a 2.6 feature, please dont merge yet

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx [hunk 1]
```diff
@@ -187,6 +189,18 @@ const InfraEnvsTable: React.FC<InfraEnvsTableProps> = ({ infraEnvs, agents }) =>
                             )
                         },
                     },
+                    {
+                        header: t('infraEnv.tableHeader.creationDate'),
+                        sort: (a: InfraEnvK8sResource, b: InfraEnvK8sResource) => {
```
Comment (jtomasek) [inline frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx:194]: IIUC this should work as well:
```
sort: 'metadata.creationTimestamp',
```

**Review by jtomasek — CHANGES_REQUESTED** [review]

File: frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx (hunk 1 above)
Comment (ammont82) [reply frontend/src/routes/Infrastructure/InfraEnvironments/InfraEnvironmentsPage.tsx:215]: To sort correctky date fields we need to do this:
       const dateTimeCellA = getDateTimeCell(a.metadata?.creationTimestamp ? new Date(a.metadata?.creationTimestamp).toString() : '-');
                            const dateTimeCellB = getDateTimeCell(b.metadata?.creationTimestamp ? new Date(b.metadata?.creationTimestamp).toString() : '-'); 
                            return compareStrings(((dateTimeCellA.sortableValue == 0) ? '':dateTimeCellA.sortableValue.toString()), ((dateTimeCellB.sortableValue == 0) ? '':dateTimeCellB.sortableValue.toString()))

**Review by jtomasek — COMMENTED** [review]
Looks good, needs rebase after https://github.com/stolostron/console/pull/1469 merges

**Review by jtomasek — APPROVED** [review]
/lgtm

**Pull request merged**